$NetBSD$

on NetBSD/amd64 9.4: getting

autoreconf: running: libtoolize --copy --force
libtoolize: putting auxiliary files in AC_CONFIG_AUX_DIR, 'tools'.
libtoolize: copying file 'tools/ltmain.sh'
libtoolize: Consider adding 'AC_CONFIG_MACRO_DIRS([m4])' to configure.ac,
 ...
 ...
loading site script './config.site'
loading build-specific script './config.site'
./configure: 4429: Syntax error: Word "blas" unexpected (expecting ")")
*** Error code 2


configure(s) generated are different on 10.99.12 and 9.4
release    line # 
--------   ------ 
10.99.12   4770   withval=$with_blas; if test "${withval}" = no; then
 9.3       4392   withval=$with_blas; R_ARG_USE(blas)

--- configure.ac	2026-01-02 17:46:48.409027336 +0900
+++ configure.ac	2026-01-02 17:46:48.409027336 +0900
@@ -33,6 +33,7 @@
 AC_INIT([R],[R_VERSION],[https://bugs.r-project.org],[R],[https://www.r-project.org])
 AC_CONFIG_SRCDIR([src/include/Defn.h])
 AC_CONFIG_AUX_DIR([tools])
+AC_CONFIG_MACRO_DIRS([m4])
 
 ### * Information on the package.
 
